change content of physics file

ducphu130894 opened the pull request:
please review this

ducphu130894 committed:
change content of physics file

ducphu130894 committed:
adding image for physics

teymourj commented on Common_Files/Physics.ipynb:
```diff
@@ -10,19 +10,53 @@
     "\n",
-    "Below is the recommended general structure to develop the Physics file for the flow of interest. Please note that we are always open to new ideas and modifying the structure below. Thus, do not limit yourself to the following structure and feel free to implement your suggestion and changes. We can modify this structure based on new feedback.\n",
+    "<img src=\"./Images/fluid1.jpg\" width=\"500\">\n",
+    "\n",
```
Add a general and descriptive explanation of the physics behind the steady and uniform flow over a rectangle.  Use/Create a simple "sketch" instead of the colorful CFD results. Note that your readers are learning CFD and have not idea at the Physics section about it.

Also discuss that you will use two different approaches (Control Volume and Dimensional Analysis) to address the physics of the flow over rectangle.

teymourj commented on Common_Files/Physics.ipynb:
```diff
@@ -10,19 +10,53 @@
+    "\n",
+    "\n",
+    "The equation for conservation of momentum is used to obtain a general understanding about the interaction between the flow and the rectangle. There are two force components acting on the rectangular body. One force component is in the stream wise direction and is called the Drag Force. The other component is perpendicular to the flow direction and is called the Lift Force (i.e. flow tries to ...
+    "\n",
```
If you can include the definition of the CV in your sketch at the beginning, you can refer to that sketch here. Consider this.

teymourj commented on Common_Files/Physics.ipynb:
```diff
@@ -10,19 +10,53 @@
+    "$$FD=f (d,μ,ρ,V)$$\n",
+    "\n",
+    "where d is the dimension of the rectangle, μ and ρ are the viscosity and density of the fluid. As there are 5 variables and 3 reference dimensions (i.e., Mass, Length, Time), two non-dimensional groups (i.e. Π groups) are needed to fully define the flow field. Performing dimensional analysis result in two following non-dimensional groups:\n",
+    "\n",
```
Correct these equations similar to my previous modifications.

teymourj commented on Common_Files/Physics.ipynb:
```diff
@@ -10,19 +10,53 @@
+    "\n",
+    "$$CD=ϕ(Re)$$\n",
+    "\n",
+    "In this equation CD, the total coefficient of the drag force due to pressure and viscous forces within the flow acting on the rectangle's surface in the stream wise direction. This coefficient is the normalized drag force component with the available momentum in the undisturbed incoming flow that could be exerted on the rectangle. The value of this momentum is calculated using ρ as the refer ...
```
is "d" diameter of the rectangle or width of it that is seeing the incoming flow?

Please confirm/correct it.

teymourj committed:
First revision of Phu submission for Physics file of F.

teymourj commented:
@ducphu130894 I just finished reviewing your submission. Overall, you did an excellent job! I really enjoyed seeing this quality of work from your side. Sentences are complete, scientific content is accurate and your general path to describe the physics was great. I can see that you used the previous templates (which is totally fine) but you made the content on your own! I am proud of you.

I have revised the content and implement multiple changes directly in to the text. Make sure you see those changes and see the points that you still need to pay attention to.

There are some required corrections and additions that I have put a comment for each of them within the body of text. You can see those comments in the above section here.

First of all, sync the content of the repo.. Then review my changes and comments. Apply the required changes and push the new version of the Physics file back on this pull request for my revision. I think after the second review we can finalize the content of the Physics file and move forward.

Keep up your great job! 👍

ducphu130894 committed:
Updated the Mesh, Set up and Post Process

I have updated most of the files, only need to finish the data for the
Post process part.

teymourj committed:
Edits on the Domain file.

teymourj commented on Common_Files/Domain.ipynb:
```diff
@@ -6,96 +6,37 @@
     "\n",
-    "The element size chosen to refine the mesh was 25mm, which will give 80 elements along the length of the rectangle and 40 elements along the width of the rectangle. This type of distribution is not ideal since a smaller element size should be assigned so that there are more than 100 elements along the length of the rectangle. However, the results were close to the experiment data conducted b ...
-    "Fig.3 shows the fully meshed body and Fig.4 shows the zoom in view of the fine mesh regions.\n",
+    "The element size chosen to refine the mesh was $25 [mm]$, which leads to 80 elements along the length and 40 elements along the width of the rectangle. Fig. 4 shows the zoom in view of the meshed region aroud the central rectangle. It is worth noting that the two slender rectangles on the left should have been coarsely meshed, because the inlet flow is uniform, thus more calculation in those ...
```
Add detail of mesh size/distribution on for the subdomains.
Add description about mesh element type and why you chose them.

teymourj committed:
Add edits to the Simulation file.

teymourj commented:
All you newly submitted files have been revised. Here is the summary:

1- There are some comments on the Physics file that needs to be implemented from previous revision. To see those comments just click on the corresponding "Show 1 Comment" button on top.
2- There is one more comment that needs implementation on the Domain file. After that, it is good to go.
3- Edits on the Simulation file has been implemented directly and that file is finalized.
4- The Post-Process file seems to need more work to complete. Finalize that and correct the format of equations and re-submit it for final edits and revision.

Make sure to Sync the repo. and pull-down the newly implemented corrections before start to implement new comments. 

Make sure not to push the checkpoint files on your new submission.

ducphu130894 committed:
Changed Physics and Domain files

ducphu130894 committed:
Changed to physics file

ardeshirj committed:
Remove ven dir

teymourj committed:
Add final edits on content and equations of the Physics file.

teymourj committed:
Recovered the defected Domain file.

teymourj commented:
@ducphu130894 This is the last chance I want to give to update this case study! 

The uploaded updates that you had were not making any sense to me! The only effective change that I could see was the rough sketch of the flow field for the Physics file. The uploaded Domain file was defected and not readable!

Since I have made the comments to modify the Physics file multiple times, but they have been disregarded, I ended up implementing those comments by myself. Now the equations are all corrected and content is finalized.

There are two things that need you updates and final inputs:
1- Details of the mesh setting on the Domain file. Look at my ONLY comment that I made on the spot of the Domain file on Sep 2. Please don't disregard that and apply the changes correctly. As I said the submitted Domain file at your last try was not readable.

Make sure to sync the repo. before start to change anything. I have changed multiple files. Also make sure not to push the checkpoint files on your new submission.

2- Finalize the content of the Post_Process file. You need to clean-up the content and equations. You also supposed to add comparison of values for the Drag coefficient.

Please make these changes and submit it to the repo. ASAP. This is taking way longer than it supposed to.

ducphu130894 committed:
Added content to the Domain files

I have added the changes to your comments on September 2.

ducphu130894 committed:
Post process finished

I have finished the post process, added graphs and conclusion

teymourj committed:
Final revise of Domain and Post_Process files.
